diff --git a/apis/python/setup.py b/apis/python/setup.py
index a1237d1..ac5e44e 100644
--- a/apis/python/setup.py
+++ b/apis/python/setup.py
@@ -100,8 +100,8 @@ def find_libtiledbvcf():
     if LIBTILEDBVCF_PATH:
         p.native_lib_install_dirs = [LIBTILEDBVCF_PATH]
 
-    libdirs = ["lib"]
-    libnames = ["libtiledbvcf.dylib", "libtiledbvcf.so"]
+    libdirs = ["lib", "bin"]
+    libnames = ["libtiledbvcf.dylib", "libtiledbvcf.so", "tiledbvcf.dll"]
     for root in p.native_lib_install_dirs:
         for libdir in libdirs:
             for libname in libnames:
