diff --git a/libtiledbvcf/cmake/Superbuild.cmake b/libtiledbvcf/cmake/Superbuild.cmake
index ced7e1f..202066f 100644
--- a/libtiledbvcf/cmake/Superbuild.cmake
+++ b/libtiledbvcf/cmake/Superbuild.cmake
@@ -51,6 +51,7 @@ set(INHERITED_CMAKE_ARGS
   -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
   -DEP_BASE=${EP_BASE}
   -DFORCE_EXTERNAL_HTSLIB=${FORCE_EXTERNAL_HTSLIB}
+  -DFORCE_EXTERNAL_SPDLOG=${FORCE_EXTERNAL_SPDLOG}
   -DFORCE_EXTERNAL_TILEDB=${FORCE_EXTERNAL_TILEDB}
   -DTILEDB_S3=${TILEDB_S3}
   -DTILEDB_WERROR=${TILEDB_WERROR}
